modules/ocf_desktop/manifests/drivers.pp

```diff
@@ -1,4 +1,8 @@
 class ocf_desktop::drivers {
+  class { 'ocf::apt::i386':
```

Comment:
Should this be moved to `init.pp`? We can only define it once even though it's needed for other desktop things like steam.